packages/@react-aria/focus/stories/FocusScope.stories.tsx

```diff
@@ -149,6 +149,27 @@ function FocusableFirstInScopeExample() {
   );
 }
 
+function IgnoreRestoreFocusExample() {
+  const [display, setDisplay] = useState(false);
+
+  return (
+    <div>
+      <button type="button" onClick={() => setDisplay(state => !state)}>
+        {display ? 'Close dialog' : 'Open dialog'}
+      </button>
+      {display &&
+        <FocusScope autoFocus restoreFocus>
```

Comment:
```suggestion
        <FocusScope restoreFocus>
```
I think we don't want autoFocus because then focus would go inside the scope and it'd become the active scope.
I'm also a little confused how to test the changes with this story, there's no visible focus and nothing says where focus currently is.